meta-ov/recipes-kernel/linux/linux-mainline/openvario.cfg

```diff
@@ -27,7 +27,7 @@ CONFIG_UEVENT_HELPER=y
 
 # CONFIG_INPUT_FF_MEMLESS is not set
 # CONFIG_INPUT_MOUSE is not set
-CONFIG_INPUT_UINPUT=m
+CONFIG_INPUT_UINPUT=y
```

Comment:
> It is needed by ts_uinput, so it's always loaded, and there's no point in making it modular.

Well, unless device has no touchscreen. But I guess we are running `ts_uinput` regardless, so in practice it is getting loaded all the time anyways.